.gitignore

```diff
@@ -6,3 +6,9 @@ cabal-dev
 cabal.sandbox.config
 .*.swp
 .*.swo
+/doc/autom4te.cache/
+/doc/config.log
+/doc/config.mk
+/doc/config.status
+/doc/configure
+/doc/happy.1
```

Comment:
I would put these in a separate `.gitignore` inside the docs subdir. How does that sound?